jetty-server/src/main/java/org/eclipse/jetty/server/SecureRequestCustomizer.java

```diff
@@ -377,4 +400,40 @@ public Object getAttribute(String name)
             return names;
         }
     }
+
+    /**
+     * Simple bundle of information that is cached in the SSLSession.
+     */
+    private static class SslSessionData
+    {
+        /**
+         * The name of the SSLSession attribute that will contain this information.
+         */
+        private static final String ATTR = SslSessionData.class.getName();
+        private final Integer _keySize;
```

Comment:
`int`, not `Integer`.

Reply:
Isn't `Integer` better here so it doesn't get boxed on every get?

Reply:
Ah yes, `Integer` everywhere - I missed the boxing when returning the attribute, and the `keySize` values are not cached in `Integer` (only caches `-128` to`127`).

Reply:
It's Integer everywhere now.